<launch>
    <node pkg="eac_pkg" type="navigation_node" name="navigation_node">
        <param name="side_color" value="0"/>
    </node>
</launch>